server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -1110,7 +1110,9 @@ private static void checkWalCanSync(ServerContext context) {
 
   private void config() {
     log.info("Tablet server starting on {}", getHostname());
-    Threads.createThread("Split/MajC initiator", new MajorCompactor(context)).start();
+    // TODO KEVIN RATHBUN running major compactions is a critical function of the TabletServer.
+    // also this thread is only created once.
+    Threads.createCriticalThread("Split/MajC initiator", new MajorCompactor(context)).start();
```

Comment:
running major compactions is a critical function of the TabletServer. also this thread is only created once.